app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/CustomNavigationView.java

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.services.android.navigation.testapp.activity.navigationui;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+
+import com.mapbox.services.android.navigation.testapp.R;
+import com.mapbox.services.android.navigation.ui.v5.NavigationView;
+
+public class CustomNavigationView extends NavigationView {
+
+  public CustomNavigationView(Context context) {
+    super(context);
+  }
+
+  public CustomNavigationView(Context context, @Nullable AttributeSet attrs) {
+    super(context, attrs);
+  }
+
+  public CustomNavigationView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+    super(context, attrs, defStyleAttr);
+  }
+
+  @Override
+  protected int getLayoutRes() {
+    return R.layout.custom_navigation_view;
```

Comment:
@devotaaabel if a developer doesn't include all of the `View`s (in addition to their custom views) needed for the base `NavigationView` in this layout XML, we will crash when initializing right?

Reply:
@danesfeder yes it will crash. The scope of this PR is to allow the developers to remove and/or modify the feedback and sound buttons, a later addition could let them exclude other views as well.